### ISSUE TITLE
Allow scout upload of cases with downsampled/external samples 

### ISSUE THREAD

**Issue by fevac** [Clinical-Genomics/cg#2279]
## Description

- Currently the upload of downsampled/external cases will fail. The reason is that the delivery report fails to be created as it requires information from LIMS and these samples are missing there.

It would be good to allow the upload of these cases by either:
- get this information from the original sample by checking `from_sample` 
- add an option to `--skip-delivery-report`
- add option `--skip-lims-info` that allows upload even if no samples are found in lims and adds NAs in the delivery report for those metrics coming from lims. 

## Suggested solution

*Write here if there is a suggested way to solve the issue*

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by fevac** [comment]
@ivadym what you think about this?

**Comment by ivadym** [comment]
There is already an issue regarding this:
https://github.com/Clinical-Genomics/cg/issues/1842

**Comment by fevac** [comment]
Great! Although inheriting information would work for downsampled samples but won't for external samples. So maybe we would also like to have a `--skip-lims-info` for such cases.

Is there any plan to do this soon?

**Comment by fevac** [comment]
I feel like `--skip-lims-info` would not take so long and this is a general solution. Then we could do another PR for the issue you linked

**Comment by ivadym** [comment]
There is already an issue regarding this (and a plan to do it, maybe we need to reprioritise it):
https://github.com/Clinical-Genomics/cg/issues/1842

**Comment by ivadym** [comment]
> I feel like `--skip-lims-info` would not take so long and this is a general solution. Then we could do another PR for the issue you linked

It should work for external samples (https://github.com/Clinical-Genomics/cg/pull/1556) and you should be able to bypass it with the `--force` flag. Let me have a look at the specific case you're talking about 🤔

**Comment by fevac** [comment]
Conclusion:
- external samples should always have lims accessions (coming from orders), so there is no need to implement anything for these.
- downsampled samples should be fixes as Beatriz suggested in her issue.
Closing in favour of https://github.com/Clinical-Genomics/cg/issues/1842
